### ISSUE TITLE
Empty css files in destination

### ISSUE THREAD

**Issue by klwemu** [zont/gulp-usemin#94]
Since a view days, Did not write into destination file when gulp.src() returns more than 1 file.   
We had used the same Gulpfile since months without trouble. But maybe the last update .. i dont know exactly

```javascript
gulp.task('test', function () {
    return gulp.src('app/*.html')  
        .pipe(usemin({
            css: [ minifyCss(), 'concat' ]
        }))
        .pipe(gulp.dest('dist/'))
});
```
In detail:
gulp.src('app/index.html')    returns only 1 File and all works fine.
gulp.src('app/in*.html')    returns 2 Files - with error!   stream.js:94 throw er; // Unhandled stream error in pipe.

**Comment by sc0ttyd** [comment]
:+1: I'm also experiencing this. Do you have a resolution?

**Comment by dtryon** [comment]
:+1:

**Comment by calledT** [comment]
@sc0ttyd this may cause by gulp-minify-css.I use v0.3.11 everything is fine, when I turn to v0.3.13, the output css files is empty.

**Comment by mofas** [comment]
encounter this bug too.

**Comment by wulfsberg** [comment]
It seems 0.3.11 fixes this bug, but unfortunately includes #105, which puts me between a rock and a hard place.

**Comment by zont** [comment]
Fixed in 0.3.14
